### DISCUSSION TITLE
how to disable "preserve in github arctic program" for all repos? or at least disable it by default?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [community/community#46847] [Repositories]
### Select Topic Area

Product Feedback

### Body

- how to disable "preserve in github arctic program" for all repos? or at least disable it by default?
- this option is only available per repo - and there seems to be no way to disable it by default.
- and also, it gets hidden for "archived" repos, which makes it even harder to disable this :(
- there is NO way to disable this while forking a repo either

![image](https://user-images.githubusercontent.com/19423063/217817126-018aee1b-4d6a-4a76-8539-f70655f33ff5.png)

---
![vivaldi_Zzy799baoC](https://user-images.githubusercontent.com/19423063/217865969-a28d7a54-a643-4c1d-ad5f-95909c92782a.png)

**Comment by mgavin** [comment]
LOL! No followup on this?

Sounds like one of those things that's either intentionally or ignorantly hidden from github users to easily configure.
